aircraft/breguet_range.py

```diff
@@ -1,8 +1,7 @@
 """Breguet range model"""
-from gpkit.shortcuts import Var, Model
+from gpkit import Variable, Model
 from gpkit.tools import te_exp_minus1
 
-
```

Comment:
I have https://github.com/scrooloose/syntastic. I remember it took me awhile to get it set up so don't start that project if you have looming deadlines (ahem)